integration_tests/integration/prisonerReport.cy.ts

```diff
@@ -401,4 +386,47 @@ context('Prisoner report - reporter view', () => {
       })
     })
   })
+  describe('review statuses', () => {
```

Comment:
It's confusing that these tests are virtually the same as the above ones but these are for `review statuses` and the other is for `test prisoners`. In all they do test all 4 statuses but it's not obvious.

The problem is that if/when these come to be modified people will just not know what it tests and therefore what the values should be.

I have made a separate comment that there isn't much point checking these statuses in int tests as they are already covered by the jest tests.